crop_coefficient.py

```diff
@@ -0,0 +1,35 @@
+import math
+import numpy as np
+import pandas as pd
+from datetime import datetime
+
+
+def crop_coefficient(date_time, planting_date, vegetation_type, vegetation_height):
+    
+    df = pd.read_excel('crop_coefficients.xlsx')
```

Comment:
Can you convert this to a `.csv` file and include that in the PR as well. It would be good to have it in the repo.